### ISSUE TITLE
When init-ing don't overwrite pre-existing files (to aid the distribution of astrum projects)

### ISSUE THREAD

**Issue by philipbelesky** [NoDivide/astrum#66]
#### Tell us about the issue you are experiencing

My use case is for distributing an npm/bower package with astrum as a dev dependency but without *without* bundling/packaging the core pages generated when creating an astrum project — ie the `index.html` and `app/` files that were auto-generated and that I've no need to modify. The package *would* include the `/components`, `data.json`, etc that are project-specific.

#### What behaviour would you expect?

It would be great if when running `astrum init folder` the init process would skip over the files it would create in `folder` it they already exist; or that it could be run with a flag that allows it to do so optionally.

#### How can this issue be replicated?

Take your astrum project, delete the `index.html` or `app/` folders, etc.

#### Screenshots / Screencast

N/A

#### Your environment

Platform: Mac OS X 
OS Version: 10.11.6
Node Version: 6.5.0
NPM Version: 3.10.3
Astrum Version: 1.7.0

#### Other

Would you be interested in a pull request that implements this along with the original behaviour? I should be able to give it a shot. Link to #23 as it assumes `init` will work in a pre-existing folder.

**Comment by RyanHavoc** [comment]
@philipbelesky Are you intending for the Astrum core files to be instantiated when you run your bundled package then? Sorry, I'm having trouble getting my head around what you're trying to achieve (it is my first day back from a weeks leave mind 😕)

**Comment by philipbelesky** [comment]
@RyanHavoc yes. So I imagine in my case (as with many use cases for Astrum) the package itself provides a compiled CSS file of styles that implement a pattern library.  However, if a user wants to edit and contribute to these styles, I've setup my `devDependencies` such that it pulls down the packages needed to do so — ie gulp, a sass compiler, a server, and of course Astrum. 

As with any other package dependency I'd like to only include its files into my package when necessary, such as when adding Astrum components or the Astrum config files. Adding the `index.html` and `app/` folder seems akin to me to adding the compiled `boostrap.css` to a project's source — it should be avoided unless doing some direct customisation. Does that make sense?

**Comment by RyanHavoc** [comment]
@philipbelesky Without the `app/` and `index.html` files though the pattern library won't work... Don't you want the pattern library to be included with your project? Once added to the repo `app/` and `index.html` are static and don't change (unless there's an Astrum update and you want to update your instance). Am I miss-understanding?

**Comment by philipbelesky** [comment]
My thinking is I don't want to include it the pattern library as it's only useful for those looking to edit the underlying styles, as opposed to just using the CSS as a drop in. Not having those files present  makes the project both slightly smaller and removes the need to update those files when Astrum does.

**Comment by schneyra** [comment]
I would really love to see that implemented!
